03/app.js

```diff
@@ -7,12 +7,61 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
+function randomNumber(min, max) {
+    return Math.round((Math.random() * (max - min)) + min);
+}
 
+function getSum(a, b, c) {
 
+    if (a > b && b > c){
+        return parseInt(a) + parseInt(b);
 
+    } else if (a > b && c > b){
+        return parseInt(a) + parseInt(c);
 
+    } else if (a < b && a < c){
+        return parseInt(b) + parseInt(c);
 
+    } else if (a < b && a >c){
+        return parseInt(a) + parseInt(b);
+    }
```

Comment:
Jeśli wiemy, że a nie jest największa to wiemy, że b i c jest - nie trzeba ponownie tego sprawdzać ;)

PS. Najłatwiej by było zrobić z tych liczb tablicę i posortować ;)